src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs

```diff
@@ -227,9 +236,15 @@ public override StandardValuesCollection GetStandardValues(ITypeDescriptorContex
                 // We need to get the enum values in this rather round-about way so we can filter
                 // out fields marked Browsable(false). Note that if multiple fields have the same value,
                 // the behavior is undefined, since what we return are just enum values, not names.
-                Type reflectType = TypeDescriptor.GetReflectionType(EnumType) ?? EnumType;
+                [UnconditionalSuppressMessage("Trimming", "IL2067:", Justification = "Trimmer does not trim enums")]
+                static Type GetTypeDescriptorReflectionType(Type enumType) => TypeDescriptor.GetReflectionType(enumType);
+
+                Type reflectType = GetTypeDescriptorReflectionType(EnumType) ?? EnumType;
+
+                [UnconditionalSuppressMessage("Trimming", "IL2070:", Justification = "Trimmer does not trim enums")]
+                static FieldInfo[]? GetPublicStaticEnumFields(Type type) => type.GetFields(BindingFlags.Public | BindingFlags.Static);
```

Comment:
If we are going to propagate this suppression throughout the stack, would it make sense to make new APIs specifically for Enums? There are other places that get the FieldInfo of enums. For example: 

https://github.com/microsoft/kiota-abstractions-dotnet/blob/9852c37689c53c33ba91dbeb8a74dfe1e555d4f8/src/RequestInformation.cs#L209-L214

Reply:
Created #100592 to track this.